src/sage/calculus/desolvers.py

```diff
@@ -27,8 +27,8 @@
   order equations, return list of points.
 
 - :func:`desolve_odeint` - Solve numerically a system of first-order ordinary
-  differential equations using ``odeint`` from `scipy.integrate module.
-  <https://docs.scipy.org/doc/scipy/reference/integrate.html#module-scipy.integrate>`_
+  differential equations using :func:`~scipy:scipy.integrate.odeint` from
```

Comment:
I'm not sure to understand the meaning of the `~`. Here we have ``` :func:`~scipy:scipy.integrate.odeint` ```, but line 1502 you use ```  :func:`scipy:scipy.integrate.odeint` ```.  Why and why is it needed ?

Reply:
The `~` gives short links, such as "desolve_system()" instead of "scipy.integrate.odeint()"

Reply:
Here I use it because the text already says from what module it comes.